qa/sc_cert_key_rotation.py

```diff
@@ -17,7 +17,7 @@
 from SidechainTestFramework.secure_enclave_http_api_server import SecureEnclaveApiServer
 from httpCalls.submitter.getCertifiersKeys import http_get_certifiers_keys
 from httpCalls.submitter.getKeyRotationProof import http_get_key_rotation_proof
-from httpCalls.submitter.getSchnorrPublicKeyHash import http_get_schnorr_public_key_hash
+from httpCalls.submitter.getKeyRotationMessageToSign import http_get_key_rotation_message_to_sign
```

Comment:
add negative case, with invalid message to sign because of:
1) wrong key type
2) wrong epoch number

Reply:
Wrong key type I added, wrong epoch number is impossible to check, because there's no response 
![image](https://user-images.githubusercontent.com/110401079/214934815-348e44c4-615d-4b2b-ae3a-0353bf0d6ba7.png)

Reply:
Wrong key type I added, wrong epoch number is impossible to check, because there's no response 
![image](https://user-images.githubusercontent.com/110401079/214934815-348e44c4-615d-4b2b-ae3a-0353bf0d6ba7.png)

Reply:
Actually key type is also checked on request level
![image](https://user-images.githubusercontent.com/110401079/214972399-5d90e33f-9073-4e02-bb40-0ab5468763ef.png)